salesforce/packagebuilder.go

```diff
@@ -289,3 +296,16 @@ func getMetaForPath(path string) (metaName string, objectName string) {
 	objectName = fileName
 	return
 }
+
+func (files *ForceMetadataFilesForType) MetaType() MetaType {
```

Comment:
What is this? Could use a docstring, not very apparent. Also, it looks like it's a function for a struct, but not sure where that struct lives. Why include it in this file? It doesn't look like it has anything to do with packagebuilder.go.